app/layout.tsx

```diff
@@ -37,17 +37,13 @@ export default function RootLayout({
   return (
     <html lang="he" dir="rtl" suppressHydrationWarning={true}>
       <body
-        className={`min-h-screen m-0 darkmode-main ${birzia.className} ${inter.variable}`}
+        className={`min-h-screen w-full flex flex-col items-center m-0 darkmode-main ${birzia.className} ${inter.variable}`}
```

Comment:
Flex is the correct approach to this issue :)

Don't need w-full since that's body's default

I think having items-center at the root will be off throwing for everything else, 
The default 'stretch' is more predictable
perhaps 'start' so others feel like they're working with a normal container, e.g. a button will behave as if it was placed inside a regular div
instead of unexpectedly placed in the middle(center) or stretching to the full available width(stretch)